include/mango_v3.hpp

```diff
@@ -125,7 +127,7 @@ struct MangoAccount {
   i80f48 deposits[MAX_TOKENS];
   i80f48 borrows[MAX_TOKENS];
   solana::PublicKey spotOpenOrders[MAX_PAIRS];
-  PerpAccount perpAccounts[MAX_PAIRS];
+  PerpAccountInfo perpAccounts[MAX_PAIRS];
```

Comment:
yep, good call, you found an inconsistency here in the upstream code. we should make this consistent in mango v4